all-imports.html

```diff
@@ -0,0 +1,12 @@
+<!--
```

Comment:
Was using this to include both the mixin and behavior in the `index.html` `iron-component-page`. I have seen this a lot on repositories like `paper-input`. I can get rid of this but what would you like linked in the `index.html`?